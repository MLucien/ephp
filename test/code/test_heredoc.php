<?

$heredoc = <<<'HERE'

This is a heredoc! :-)
just as it should be

HERE;

print $heredoc;

$heredoc = <<<END

And another here doc, with
a tramp...
HERE;

END;

print $heredoc;

$who = "World";
$times = 10;
$heredoc = <<<VARS

Hello $who {$times} times (originally ${times})

VARS;

print $heredoc;

