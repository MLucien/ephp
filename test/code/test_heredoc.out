
This is a heredoc! :-)
just as it should be

And another here doc, with
a tramp...
HERE;

Hello World 10 times (originally 10)
